snyk-sdk/src/main/java/io/snyk/sdk/model/NotificationSettings.java

```diff
@@ -0,0 +1,34 @@
+package io.snyk.sdk.model;
```

Comment:
I think I need an explanation on what this file is about

Reply:
This file is a java mapping class for json (endpoint https://snyk.docs.apiary.io/#reference/users/user-notification-settings/get-notification-settings).